src/app/home/home.component.html

```diff
@@ -60,15 +60,15 @@ <h2>Simple</h2>
 
     <p>
       With smart coding hints from IntelliSense, Small Basic makes writing code
-      a breeze.
+      a breeze. <a href=https://social.technet.microsoft.com/wiki/contents/articles/14114.small-basic-the-unique-features.aspx#Simple target="_blank">Learn more</a>.
```

Comment:
Accessibility: link text alone should reflect the link destination. Do not use "click me" or "learn more"